src/commands/Fun/initial.js

```diff
@@ -0,0 +1,12 @@
+exports.run = function(bot, msg, args) {
+    if (args.length < 1) {
+        throw new Error('You must input text to be transformed.');
```

Comment:
Just a stylistic thing, but I usually don't create error objects. I just throw a string. I suppose this isn't too important, but it might be nice to retain consistency throughout the code.